content/influxdb/v2.1/reference/cli/influxd/downgrade.md

```diff
@@ -0,0 +1,46 @@
+---
+title: influxd downgrade
+description: >
+  The `influxd downgrade` command downgrades the metadata schema used by
+  `influxd` to match the schema of an older release.
+menu:
+  influxdb_2_1_ref:
+    parent: influxd
+weight: 201
+related:
+  - /influxdb/v2.1/upgrade/downgrade/
+---
+
+Use the `influxd downgrade` command to downgrade the metadata schema used by
+`influxd` to match the metadata schema of a older release.
+
+InfluxDB does not guarantee backwards-compatibility with older releases in its embedded metadata stores.
+Attempting to start an older `influxd` binary with a BoltDB or SQLite file that
+has been migrated to a newer schema will result in a startup error.
```

Comment:
"migrated" refers to a data migration. In CI workflows, it's common to update data schemas. To accomplish this gracefully, you have to write a script that adds/updates the existing data to conform to the new schema. These scripts are known as "migrations."

Because we change the schema of the metadata between versions, we include "migrations" built into `influxd`. The migrations that automatically run are only forward-moving (run when a newer version is used). `influxd downgrade` provides backwards migrations; migrations that modify the metadata to fit the schema of a specific older version.